packages/shared/components/organisms/ProposalCard.svelte

```diff
@@ -32,10 +32,12 @@
 
 <proposal-card
     on:click={handleProposalClick}
-    class="flex flex-col p-6 border border-solid border-gray-200 dark:border-transparent rounded-xl cursor-pointer h-32
-    {proposal.status === ProposalStatus.Ended ? 'bg-transparent' : 'bg-white dark:bg-gray-850'}"
+    on:keydown={(e) => e.key === 'Enter' && handleProposalClick()}
```

Comment:
This doesn't work for me

Reply:
It it working here, you need to tab until the focus is on one of the proposal cards and then press `Enter`